examples/npb-3.4.3/campaign_npb.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+# Copyright (C) 2024 Vrije Universiteit Brussel. All rights reserved.
+# SPDX-License-Identifier: MIT
+"""
+Minimal example to run NPB benchmark.
+"""
+
+
+from typing import Iterable, List
+
+from npb import NPBBench
+
+from benchkit.campaign import CampaignCartesianProduct, CampaignSuite
+from benchkit.utils.dir import caller_dir
+
+
+def npb_campaign(
+    benchmark_duration_seconds: int = 3,
```

Comment:
```suggestion
    benchmark_duration_seconds: int = 0,
```
The NPB campaign does not use the benchmark_duration_seconds -- the benchmark does not allow time to be configured

Reply:
I just realized that the npb_campaign defined in the benchkit support itself (and not the example) has the same issue.
Could you also change that to using `None`?

Reply:
done